serverless/serverless-getting-started.adoc

```diff
@@ -13,23 +13,30 @@ include::modules/technology-preview.adoc[leveloffset=+1]
 
 [id="how-it-works_{context}"]
 == How {ServerlessProductName} works
-Developers on {ServerlessProductName} can use the provided Kubernetes-native APIs, as well as familiar languages and frameworks, to deploy applications and container workloads.
-For information about installing {ServerlessProductName}, see xref:../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[Installing OpenShift Serverless].
 
-{ServerlessProductName} on {product-title} enables stateful, stateless, and serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
+Developers on {ServerlessProductName} can use the provided Kubernetes native APIs, as well as familiar languages and frameworks, to deploy applications and container workloads.
+
+{ServerlessProductName} on {product-title} enables stateless serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
 
 {ServerlessProductName} is based on the open source Knative project, which provides portability and consistency across hybrid and multi-cloud environments by enabling an enterprise-grade serverless platform.
 
-[id="knative-eventing_{context}"]
-== Knative Eventing
-A developer preview version of Knative Eventing is available for use with {ServerlessProductName}. However, this is not included in the {ServerlessOperatorName} and is not currently supported as part of this Technology Preview. For more information about Knative Eventing, including installation instructions and samples, see the link:https://openshift-knative.github.io/docs/docs/index.html[Knative Eventing on {product-title}] documentation.
+// [id="serverless-ha_{context}"]
+// == High availability in {ServerlessProductName}
+
+// Active/passive high availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs.
+// In a HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
```

Comment:
s/a HA/an HA ?

Reply:
I don't really think it makes sense to use "an" here, since "high" doesn't have a silent H, e.g. it would be "an high availability" which doesn't sound right to me? I think "an" before a H word is more to do with sound than a hard and fast rule?

Reply:
+1 to using "an HA." You're supposed to use the article that matches how you'd pronounce the initialism, not the expanded phrase.